src/server/pages.rs

```diff
@@ -64,22 +64,43 @@ pub fn load_file(file_name: &str) -> String {
     match file_name {
         "" | "index.html" => std::include_str!("../html/index.html").into(),
         "vue.js" => std::include_str!("../html/vue.js").into(),
+        "webrtc/index.html" => std::include_str!("../html/webrtc/index.html").into(),
```

Comment:
This function also needs to be updated to load files at runtime that is not the cargo folder, since we are going to download the webrtc files that are incompatible with our license.